docs/user_manual/projects.md

```diff
@@ -0,0 +1,235 @@
+all of the projects that you have access to in MISO. Clicking on the project's name, alias, or short name in the table
+will take you to the Edit Project page, where you can see more project details, as well as all of the items associated
+with the project. The toolbar at the top of the table includes an "Add" button for creating new projects.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Adding Projects" %}
+
+To create a new project, click the "Add" button in the toolbar near the top of the Projects list page. This will take
+you to the Create Project page, where you can enter the project details. Choose a descriptive project alias. This is
+the long name for the project. The short name should be a short, alphanumeric code for the project. Depending on your
+site's configuration, project short name may or may not be required. Be sure to also select an accurate project status.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign subsub = 1 %}
+{% include userman-heading.md section=page.section sub=sub subsub=subsub title="Reference Genome" %}
+
+Reference genome refers to a model genome that sequenced data should be aligned to. This should indicate the primary
+genome used for the project if there are multiple. The default scientific name used for samples will depend on the
+reference genome set on the project. For example, if the project's reference genome is "Human hg19 random," then
+samples created in the project will have a default scientific name of "Homo sapiens."
+
+{% include userman-toplink.md %}
+
+
+
+{% assign subsub = subsub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub subsub=subsub title="Default Targeted Sequencing" %}
+
+Default Targeted Sequencing is a project-level property. Whichever targeted sequencing is selected here will be the
+default value for all library dilutions created within the project. Note, however, that a library dilution's targeted
+sequencing options depend on the library kit used on the library. If the default targeted sequencing value is not
+compatible with the library kit, it will not be selected automatically, nor will it be available for selection.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign subsub = subsub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub subsub=subsub title="Permissions" %}
+
+The "Permissions" section of the Create Project page contains several options for controlling who has access to the
+project. These permissions will also affect the project's samples, studies, and other related items.
+
+The project's owner is the person who should have full control over the project. They will be ranted full read/write
+access to the project and all related items regardless of the other permissions set on the project.
+
+If the checkbox beside "Allow all internal users access?" is checked, then all users with the Internal role will also
```

Comment:
`ROLE_INTERNAL` role, instead of `Internal` role?

Reply:
This is user documentation, and the role is called "Internal" in the UI (Edit User page)